app/src/main/java/com/monkeyteam/chimpagne/model/location/LocationViewModel.kt

```diff
@@ -0,0 +1,23 @@
+package com.monkeyteam.chimpagne.model.location
+
+import android.Manifest
+import android.content.Context
+import android.content.pm.PackageManager
+import androidx.core.app.ActivityCompat
+import com.google.android.gms.location.LocationServices
+
+class LocationViewModel(myContext: Context) {
```

Comment:
Why do you call this a view model if it doesn't store any state and doesn't even extends the ViewModel class ?

Reply:
It does store the state of fusedLocationClient

Reply:
it's still not a view model, you do not extend the VIewModel class